react-app/src/components/Result.js

```diff
@@ -1,7 +1,45 @@
 import React from 'react';
 import { Accordion } from 'react-bootstrap';
 
+const _ = require('lodash');
+const fhirpath = require('fhirpath');
+
 function Result(props) {
+  function countResources() {
+    const resourceList = [];
+    // iterate through resources array, checking the resourceType of each entry
+    // if (new type),
+
+    const allResourceTypesPath = 'Bundle.descendants().resource.resourceType';
+    const uniqueResourceTypes = _.uniq(fhirpath.evaluate(props.bundle, allResourceTypesPath));
+
+    const countThisResource = (resourceType) =>
```

Comment:
DP2 - Does this need to live above the forEach loop? Could we just create this string within the body of the loop?  It's was a little confusing that we have a function that returns a string instead of just having a string that uses string interpolation in the forEach loop

Reply:
This is function is a modified version of the getResourceCountInBundle function from /mcode-extraction-framework/src/helpers/fhirUtils.js. I deleted it from my code and it works fine. If you want I could make the same change in the original function and submit a pull request.

Reply:
I don't think you need to worry about making the change over there, making the change here is sufficient for this task.